content/backend/graphql-python/2-queries.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Queries
+description: Creating Your First Queries
+---
+
+### About the Django structure
+It's important to understand how the Django project structure works before moving on.
+
+On the last chapter, you created a *Django Project*, which holds everything related to your application. However, Django separates the project into *apps*. Think about apps as a[separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns). You will have two apps, one for Users and one for the Links.
+
+Another essential decision is about the database. Django works out of the box with [SQLite](https://www.sqlite.org/) – a file based database – which you will use. On production systems, you should have a more robust database, such as [PostgreSQL](https://www.postgresql.org/). 
+
+### Creating the Links app
+
+<Instruction>
+
+On the project's root (next to the `manage.py` file), create the `links` app:
+
+```bash
+python manage.py startapp links
+```
+
+</Instruction>
+
+It will create the links app and some default files inside it.
+
+With an app in place, you need to define a Model – the layer between Django and the database.
+
+<Instruction>
+
+On the `links/models.py` file, add the following content:
+
+```python(path=".../graphql-python/hackernews/links/models.py")
+class Link(models.Model):
+    url = models.URLField()
+    description = models.TextField(null=True, blank=True)
+```
+
+</Instruction>
+
+<Instruction>
+
+Lastly, configure Django to use the new `links` app on the `hackernews/settings.py` file:
+
+```python(path=".../graphql-python/hackernews/hackernews/settings.py")
+INSTALLED_APPS = (
+    # After the graphene_django app
+    'links',
+)
+```
+
+</Instruction>
+
+<Instruction>
+
+Create the database tables:
+
+```bash
+python manage.py makemigrations
+python manage.py migrate
+```
+
+</Instruction>
+
+<Instruction>
+
+Enter the Django shell with the command `python manage.py shell` and create some links:
+
+```bash
+from links.models import Link
+Link.objects.create(url='https://www.howtographql.com/', description='The Fullstack Tutorial for GraphQL')
+Link.objects.create(url='https://twitter.com/jonatasbaldin/', description='The Jonatas Baldin Twitter')
+```
+
+### Creating your first Type and Schema
+In GraphQL, a *Type* is an object that may contain multiple *Fields*. Each field is calculated through *Resolvers*, that returns a value. A collection of types is called a *Schema*. Every schema has a special type called *Query* for getting data from the server and *Mutation* for sending data to the server.
```

Comment:
I like that you're highlighting all the important parts in italics! They should be be written starting with a lowercase letter though.